### ISSUE TITLE
Number of epoch to trainnig

### ISSUE THREAD

**Issue by evertonaleixo** [junyanz/pytorch-CycleGAN-and-pix2pix#742]
Hello, 

I was using pix2pix to get style of one environment and pass to other, e.g., a land with snow to be converted to a land without snow.

I have a database with more or less 8000 images from 101 groups. I was selecting one image of each group to be the vanilla image and the network should be able to learn how to convert it in other images of same group.

After training more than 500000 epoch the result is as bellow.
The output loss was:

[Epoch 604000] [D loss: 0.000411, acc: 100%] [G loss: 1381.413574]

Can you give me some help?

![591000_generated](https://user-images.githubusercontent.com/6534458/63597641-f569fa00-c58b-11e9-8acf-0d45327fa404.jpg)
![591000_test_A](https://user-images.githubusercontent.com/6534458/63597642-f6029080-c58b-11e9-82d4-39000f0f7b13.jpg)
![591000_test_B](https://user-images.githubusercontent.com/6534458/63597643-f6029080-c58b-11e9-9713-12ddef0c92ba.jpg)

**Comment by junyanz** [comment]
Did you use this repo? The logging style `[Epoch 604000] [D loss: 0.000411, acc: 100%] [G loss: 1381.413574]` does not look like this repo's style. It seems that your D is too strong and G is too weak.

**Comment by evertonaleixo** [comment]
Just inspired, I translate the code to keras, but using the same archtecture and hyper-parameters.

Do you have some tip to improve the equilibrium in D and G loss?
